### PR TITLE
Initial corrections and add action

### PR DESCRIPTION
**Pull request by QilongTang**
- Add action and modify action to build on node v20
- Correct a type from webside to website
- Update package.json

### CODE REVIEW

File: tests/e2e.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test.describe('SplashScreen', () => {
```
Comment (QilongTang) [inline tests/e2e.test.js:4]: @dnenov This needs update I think

Comment (dnenov) [reply tests/e2e.test.js]: Yes, just updated it and added the playwright config which should handle the server start in the pipeline. Working through the failures one by one, but getting there, @QilongTang !

Comment (dnenov) [reply tests/e2e.test.js]: OK, all passed now, the build is working!

**Review by dnenov — APPROVED** [review]
Fixed all pipeline issues, building fine now.

**Pull request merged**